API/Chapters.md

```diff
@@ -53,10 +53,8 @@ The endpoint URL may be defined by the chapter provider. The Podcast Hosting ser
 Parameters:
 - `rss`: RSS feed URL
 - `guid`: New Episode GUID
-- `enclosure_url`: New episode enclosure url
```

Comment:
I'll pull the RSS feed and parse it so `rss` parameter is needed. The `guid` will allow me to identify which episode to return the dashboard and chapters json link for. I don't think I need the `enclosure_url` for anything? Let me know if I'm missing something.

Reply:
`enclosure_url` is here so that you can get any info you need (duration, …)